pallets/transaction-storage/src/lib.rs

```diff
@@ -120,6 +120,15 @@ pub struct TransactionInfo {
 	block_chunks: u32,
 }
 
+impl TransactionInfo {
+	/// Get the number of total chunks.
+	///
+	/// See the `block_chunks` field of [`TransactionInfo`] for details.
+	pub fn total_chunks(txs: &[TransactionInfo]) -> u32 {
```

Comment:
@raymondkfcheung please, use ChunkIndex as in Polkadot-SDK and to do that, please merge this branch: https://github.com/paritytech/polkadot-bulletin-chain/tree/bko-bump to your PR (it just updates Polkadot-SDK version with ChunkIndex), we will just merge your PR with this bump also

```suggestion
	pub fn total_chunks(txs: &[TransactionInfo]) -> ChunkIndex {
```